src/one_dragon/envs/git_service.py

```
@@ -1,454 +1,748 @@
+            latest_beta: str | None = None
+
+            for version in versions:
+                if '-beta' in version:
+                    if latest_beta is None:
+                        latest_beta = version
+                else:
+                    if latest_stable is None:
+                        latest_stable = version
+                        break
+
+            return latest_stable, latest_beta
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)